### PR TITLE
doc: add content describing OpenStack services. Minor proofing.

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

File: doc/source/user/select-openstack-services.rst [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+   172.16.1.100 image.openstack.svc.cluster.local
+   172.16.1.100 volume.openstack.svc.cluster.local
+   172.16.1.100 compute.openstack.svc.cluster.local
+   172.16.1.100 network.openstack.svc.cluster.local
+   172.16.1.100 dashboard.openstack.svc.cluster.local
+   172.16.1.100 nova-novncproxy.openstack.svc.cluster.local
+   172.16.1.100 orchestration.openstack.svc.cluster.local
+
+You can access OpenStack service public endpoints using the host names listed
+in the /etc/hosts directory. For example, access OpenStack Horizon (dashboard)
```
Comment (toabctl) [inline doc/source/user/select-openstack-services.rst]: `/etc/hosts` is a file, not a directory

File: doc/source/user/select-openstack-services.rst [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
 Glance, Heat, Horizon, Keystone, Neutron, Nova.
 
-You can decide which services to deploy by:
+SUSE Containerized OpenStack deployment will automatically add the following
+host rules to the /etc/hosts file on the deployer:
 
-TODO.
+.. code-block:: console
+
+   172.16.1.100 identity.openstack.svc.cluster.local
```
Comment (toabctl) [inline doc/source/user/select-openstack-services.rst]: I guess the IPs are just examples? Maybe mention that? Or is that really hardcoded?

File: doc/source/user/select-openstack-services.rst [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
-================================
+=========================
+Select OpenStack Services
+=========================
 
 SUSE Containerized OpenStack currently deploys OpenStack Cinder,
 Glance, Heat, Horizon, Keystone, Neutron, Nova.
 
-You can decide which services to deploy by:
+SUSE Containerized OpenStack deployment will automatically add the following
```
Comment (jgu17) [inline doc/source/user/select-openstack-services.rst]: this paragraph is about how to access deployed openstack services. maybe belong to the verify/user openstack section in the deployment guide?

File: doc/source/user/select-openstack-services.rst (hunk 1 above)
Comment (jgu17) [reply doc/source/user/select-openstack-services.rst]: Yep, file

File: doc/source/deployment/deploy.rst [hunk 4]
```diff
@@ -58,6 +58,35 @@ To check the deployment progress of the Openstack services:
 
   kubectl get po -n openstack
 
+OpenStack Services
++++++++++++++++++++++++++
```
Comment (jesusaurus) [inline doc/source/deployment/deploy.rst]: Shouldn't these `+` chars stop at the final `s` above?

File: doc/source/user/select-openstack-services.rst (hunk 2 above)
Comment (jesusaurus) [reply doc/source/user/select-openstack-services.rst]: I agree, we should mention that this is an example IP address

File: doc/source/user/select-openstack-services.rst (hunk 1 above)
Comment (csymons-suse) [reply doc/source/user/select-openstack-services.rst]: done fixed dat

File: doc/source/user/select-openstack-services.rst (hunk 2 above)
Comment (csymons-suse) [reply doc/source/user/select-openstack-services.rst]: IPs changed to correspond with other IP address examples

File: doc/source/user/select-openstack-services.rst (hunk 3 above)
Comment (csymons-suse) [reply doc/source/user/select-openstack-services.rst]: Added to deploy guide. Left in this location as discussed in the standup. The deployment is an OpenStack installation, the user should know what they're getting.

File: doc/source/deployment/deploy.rst (hunk 4 above)
Comment (csymons-suse) [reply doc/source/deployment/deploy.rst]: Yes. done. Apparently I dozed off.

File: doc/source/deployment/deploy.rst [hunk 5]
```diff
@@ -58,6 +58,35 @@ To check the deployment progress of the Openstack services:
...
   kubectl get po -n openstack
 
+OpenStack Services
+++++++++++++++++++
+
+SUSE Containerized OpenStack deploys OpenStack Cinder, Glance, Heat, Horizon,
+Keystone, Neutron, and Nova.
+
+SUSE Containerized OpenStack deployment automatically adds the following host
+rules to the /etc/hosts file on the deployer as in the following example:
```
Comment (jesusaurus) [inline doc/source/deployment/deploy.rst]: `[...] adds the following host rules [...] as in the following example:` is kinda clunky and makes it hard to tell that the IP addresses are example addresses. I think we should remove the first occurrence of "the following" to give us `adds host rules [...] as in the following example:`

Comment (csymons-suse) [reply doc/source/deployment/deploy.rst]: fixed

File: doc/source/user/select-openstack-services.rst [hunk 6]
```diff
@@ -1,6 +1,6 @@
-====================================
-Select OpenStack services to install
-====================================
+=========================
+Select OpenStack Services
```
Comment (jgu17) [inline doc/source/user/select-openstack-services.rst]: You called it manage openstack service in the PR to 1.0.

Comment (csymons-suse) [reply doc/source/user/select-openstack-services.rst]: fixed

**Pull request merged**